### ISSUE TITLE
Unable to load OData adapter from Assembly Simple.OData.Client.V3.Adapter

### ISSUE THREAD

**Issue by scouser3008** [object/Simple.OData.Client#102]
Hi,

I've noticed a bit of deal breaking bug with version 4.4.0 of Simple OData Client:

```csharp
namespace ODataClientPlayground
{
    public class ODataRunner
    {
        private string _endpoint;

        public ODataRunner()
        {
            _endpoint = "http://packages.nuget.org/v1/FeedService.svc/";
        }

        public ODataRunner(string endpoint)
        {
            _endpoint = endpoint;
        }

        public void Run()
        {
            ODataClient oClient = new ODataClient(_endpoint);
            var result = oClient.For("Packages").FindEntriesAsync().Result;
            return;
        }
    }
}
```

Throws a System.Aggregate Exception when run, if you dig into the Stack you can see the buried error: 'Unable to load OData adapter from Assembly Simple.OData.Client.V3.Adapter' then finally buried in that is: "Could not load file or assembly 'Microsoft.Data.OData, Version=5.6.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)":"Microsoft.Data.OData, Version=5.6.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"

I was only able to get these errors by pulling the sample code into a separate project, but I was having the same issue with both the v3 and .Net40 packages.  Reverting my SimpleODataClient version to 4.3.0 fixes this issue.  

Has there been a change to the API that I'm missing?  Perhaps my usage is incorrect for 4.4.0?

**Comment by object** [comment]
Simple.OData.Client is using a newer version of Microsoft ODataLib components, in particular the version 5.6.4 of Microsoft.Data.OData, Microsoft.Data.Edm and System.Spatial, but you should be able to bind it a previous ones if you need. Can you check your application configuration files? Sometimes there are some assembly binding redirect instructions that refer to a different version.

**Comment by scouser3008** [comment]
You're right, my project App.Config has versions 0.0.0 -> 5.6.3 for the
ODataLib components.  This playground project was created from scratch
however and I used the PM Console to install the SimpleODataClient, is the
package still configured to load 5.6.3 by anaychance?

Kind Regards,
Antony

**Comment by object** [comment]
Hm, not good, I will take a look at it. In the meantime you can update Microsoft OData packages from NuGet console, so you will get 5.6.4.

**Comment by object** [comment]
Fixed in 4.5, available from MyGet feed until the release: https://www.myget.org/feed/Packages/simple-odata-client
